R/update-version.R

```diff
@@ -55,3 +54,48 @@ get_date <- function() {
   author_time <- parsedate::parse_iso_8601(Sys.getenv("GIT_COMMITTER_DATE"))
   as.Date(author_time)
 }
+
+fledge_bump_version <- function(desc, which) {
+  if (which %in% c("patch", "minor", "major", "dev")) {
+    suppressMessages(desc$bump_version(which), classes = "descMessage")
+    return(desc)
+  }
```

Comment:
We should do this ourselves, because we want to error:

- in "dev", when dev >= 9999
- in "patch", when patch >= 99
- in "minor", when minor >= 99